AppSettings/AppSettings/ISettingLoader.cs

```diff
@@ -10,6 +12,14 @@ public interface ISettingLoader
         /// </summary>
         /// <param name="settingKey">The key of the setting to load</param>
         /// <returns>The value</returns>
-        string Load(string settingKey);
+        string GetSetting(string settingKey);
```

Comment:
Why change the name of the method in this interface? We need a good reason to make a breaking change.